src/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -554,9 +554,6 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
 
         @Override
         public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
```

Comment:
Part of the interface, but perhaps all implementations do not use `stack`. But, param removal can be relatively complex from a diff point of view - best left to a dedicated PR IMO rather than incur the scope creep here